fix(license) fix instructions for license (#1862)
Kong/docs.konghq.com: app/enterprise/1.3-x/deployment/access-license.md

Click any file you would like to download.

## Programmatically accessing your license file

For programmatic access you'll need 3 pieces of information:

 - username (was provided to by email by your CSE)
 - repository name (the Bintray repository, visible in the GUI, usually named after your company name)
 - the Bintray API key for the account (see below)

To get the API key follow these steps:

>  - [Log into Bintray](https://bintray.com/login?forwardedFrom=%2Fkong%2F)

Reviewer: ```suggestion
 1. [Log into Bintray](https://bintray.com/login?forwardedFrom=%2Fkong%2F)
```

>  - Open the Profile settings (click the username at the top-right and select "Edit Profile")

Reviewer: ```suggestion
 2. Open the Profile settings (click the username at the top-right and select "Edit Profile")
```

>  - On the left select "API Key", and provide your password again

Reviewer: ```suggestion
3.  On the left select "API Key", and provide your password again
```

> To access the license try this:

Reviewer: ```suggestion
4. Access the license:
```
I'd remove 'try' from everywhere.

```bash

> BINTRAY_USERNAME="your_user_name@kong" && \

Reviewer: if you're including the `export` command in the last step, might make sense to include it here as well.

Author: you don't want to export those, only the final one imo

> BINTRAY_USERNAME="your_user_name@kong" && \

Reviewer: Not sure why this includes `@kong`, but I think this would be confusing for users - I'd interpret this as "username"+"@kong". I'd recommend removing it.

BINTRAY_REPO="your_repo_name" && \
BINTRAY_APIKEY="your_api_key" && \
curl -L -u"$BINTRAY_USERNAME:$BINTRAY_APIKEY" "https://kong.bintray.com/$BINTRAY_REPO/license.json"
```

> If successful it will display the downloaded license.

Reviewer: ```suggestion
If successful, it will display the downloaded license.
```

> To download the license file (as `license.json`) use this command:

Reviewer: ```suggestion
5. To download the license file (as `license.json`), use this command:
```

```bash
curl -L -u"$BINTRAY_USERNAME:$BINTRAY_APIKEY" "https://kong.bintray.com/$BINTRAY_REPO/license.json" -o license.json
```

> To export the license as an environment variable try:

Reviewer: ```suggestion
6. Export the license as an environment variable:
```

Click any file you would like to download.

## Programmatically accessing your license file

For programmatic access you'll need 3 pieces of information:

 - username (was provided by email by your CSE)
 - repository name (the Bintray repository, visible in the GUI, usually named after your company name)
 - the Bintray API key for the account (see below)

To get the API key follow these steps:

> 1. [Log into Bintray](https://bintray.com/login?forwardedFrom=%2Fkong%2F)

Reviewer: ```suggestion
1. [Log into Bintray](https://bintray.com/login?forwardedFrom=%2Fkong%2F).
```

> 2. Open the Profile settings (click the username at the top-right and select "Edit Profile")

Reviewer: ```suggestion
2. Open the profile settings: click or hover over the username at the top right and select **Edit Profile**.
```

> 3. On the left select "API Key", and provide your password again

Reviewer: ```suggestion
3. On the left select **API Key**, and if prompted, provide your password again. Click the copy icon to copy the key to your clipboard. 
```
Adding "if prompted" since I've never gotten the prompt - might be showing up cause of some security settings on your account?

> To Access the license:

Reviewer: ```suggestion
To access the license:
```

```bash
BINTRAY_USERNAME="your_user_name" && \
BINTRAY_REPO="your_repo_name" && \
BINTRAY_APIKEY="your_api_key" && \
curl -L -u"$BINTRAY_USERNAME:$BINTRAY_APIKEY" "https://kong.bintray.com/$BINTRAY_REPO/license.json"
```
If successful, it will display the downloaded license.

> To use the liicense either as a file or as a variable replace the last command with either of these examples:

Reviewer: ```suggestion
To use the license either as a file or as a variable, replace the last command with either of these examples:
```

> - To download the license file as `license.json`):

Reviewer: ```suggestion
- To download the license file as `license.json`:
```
